packages/application-config/test/json-schema.spec.js

```diff
@@ -152,6 +152,63 @@ describe('invalid configurations', () => {
       `"/oAuthScopes/manage/0 must match pattern \\"manage_(.*)\\""`
     );
   });
+  it('should validate that "additionalOAuthScopes" is an array', () => {
```

Comment:
nit: Maybe another test to validate the additional oauth scopes object must contain the three required properties? (`name`, `view` and `manage`).

Reply:
Thanks! Considering that we already have a test verifying that `name` is provided, and having in mind that `view` and `manage` take `[]` as default, I guess we have that covered as I see it

Reply:
Yes 👍, if `view` and `manage` have `[]` as default, with the test for `name` provided that I see, I guess that is covered.